Formula/git-delete-merged-branches.rb

```diff
@@ -43,7 +43,9 @@ def install
     clone = testpath/"clone"
 
     cd origin do
-      system "git", "init"
+      system "git", "init", "-b", "master"
```

Comment:
Looks like older `git` don't support `-b`. Could try `git config --global` instead if it writes to local `testpath/.gitconfig` or modify branch name `git branch -M master`